pyproject.toml

```diff
@@ -49,7 +47,7 @@ zip-safe = true
 license-files = ["COPYRIGHT", "LICENSE"]
 
 [tool.setuptools.package-data]
-"lsst.daf.butler_migrate" = []
+"lsst.daf.butler_migrate" = ["py.typed", "cli/resources.yaml"]
```

Comment:
Ah. Right. Hmm. I don't think any of the pip installable variants work with the butler cli plugins since nothing is setting the environment variable and for `pip install` we need to be using a different technique for finding these plugins using entry points. Regardless, if pipe_base isn't even including the file in the tar ball that pip installs then it will never work and needs to be fixed.